org.armagetronad.ArmagetronAdvanced.json

```diff
@@ -31,7 +28,8 @@
             "config-opts": [
                 "--disable-games",
                 "--disable-uninstall",
-                "progname=org.armagetronad.ArmagetronAdvanced"
+                "progname=org.armagetronad.ArmagetronAdvanced",
+                "CXXFLAGS=-DUSER_DATA_DIR='\"$${XDG_DATA_HOME}/game\"' -DUSER_CONFIG_DIR='\"$${XDG_CONFIG_HOME}/game\"' -DAUTORESOURCE_DIR='\"$${XDG_CACHE_HOME}/resource\"'"
```

Comment:
This is quite nonstandard. Apps usually use their name as a directory inside XDG DATA/CONFIG/CACHE directories.

So, for example, Armagetron could use:

`${XDG_DATA_HOME}/armagetronad`, `${XDG_CONFIG_HOME}/armagetronad` and `${XDG_CACHE_HOME}/armagetronad`